packages/server/src/models/EventModel.test.ts

```diff
@@ -37,4 +37,30 @@ describe('EventModel', () => {
 		expect(latest.id).toBe(allEvents[1].id);
 	});
 
+	test('should delete events older than a week', async () => {
+		const now = Date.now();
+		const aWeekAgo = now - Week;
+		jest.useFakeTimers();
+
+		for (const difference of [-10, -5, 0, 5, 10]) {
+			jest.setSystemTime(aWeekAgo + difference);
+			await models().event().create(EventType.TaskStarted, 'deleteExpiredTokens');
+		}
+
+		const allEvents = (await models().event().all());
+		expect(allEvents.length).toBe(5);
+
+		jest.setSystemTime(now);
+		await models().event().deleteOldEvents();
+
+		const remainingEvents = (await models().event().all());
+		expect(remainingEvents.length).toBe(3);
+
+		for (const event of remainingEvents) {
+			expect(event.created_time).toBeGreaterThanOrEqual(aWeekAgo);
+		}
+
+		jest.useRealTimers();
+	});
```

Comment:
If possible please also add another for:

- When there's 0 event in the database
- When the TTL means no event would be deleted

Reply:
Sure, done.